examples/tensorflow-resnet/README.md

```diff
@@ -0,0 +1,187 @@
+# Introduction
+
+This is a Gravity Cluster Image example using the TensorFlow ResNet (Deep Residual Network) server helm chart (2.0.13) that serves a machine learning (ML) model.  The default model is 
+for image classification.  The model is pulled from a public URL specified in the [values.yml](./resources/charts/tensorflow-resnet/values.yml)
+and can be configured to other models.  See the [TensorFlow](https://www.tensorflow.org/) website for more information on using ResNet models.
+
+# Example input and output
+
+The TensorFlow ResNet model has 1000 classifications for living creatures and objects.  Here is an example Ostrich graphic image the model will process and provide a tensor classification.
+
+![Ostrich](./sampleImages/ostrich.jpg)
+
+
+```bash
+Welcome to the Bitnami tensorflow-resnet container
+Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-tensorflow-resnet
+Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-tensorflow-resnet/issues
+
+calling predict using file: /sampleImages/ostrich.jpg  ...
+call predict ok
+outputs size is 2
+the result tensor[0] is:
+10
+the result tensor[1] is:
+[4.90122887e-10 1.7140348e-09 7.5265244e-10 9.62422142e-10 7.78775e-10 8.07670708e-10 1.95118852e-10 6.64805322e-10 1.48357947e-06 6.94784558e-06...]...
+Done.
+
+# Tensor classification retrieval example
+Tensor result id retrieve: 10
+Classification name: ostrich
+
+
+```
+
+## Building Cluster Image
+To construct the Tensorflow-ResNet Cluster Image with a dependency-free .tar file use this command.  You can then deploy the Tensorflow-ResNet as a self-contained, truly portable application for your preferred infrastructure. 
+```bash
+tele build -o tensorflowresnet.tar tensorflow-resnet/resources/app.yaml
+```
+
+Further details on installing Gravity Cluster Images is available [here](https://gravitational.com/gravity/docs/installation/). 
+
+
+## Building Application
+In addition to Cluster Images, Gravity supports packaging application helm charts as self-contained application images. The application is then deployable in the same manner as helm charts to Gravity clusters. Further information on application packaging deployment is available [here](https://gravitational.com/gravity/docs/catalog/).
+```bash
+tele build -o tensorflowresnet.tar tensorflow-resnet/resources/charts/tensorflow-resnet
+```
+
+## Deployment Configuration
+The Tensorflow-ResNet application runs in the default configuration with two ports, 8500, 8501.  The 8500 port is used for client image classification requests and configured available on a NodePort of 30090 by default.  
```

Comment:
Not sure whether it's worth mentioning the second port since you're not explaining what's for.
I'd just leave the first port:
```
The Tensorflow-ResNet application uses the container port 8500 which is available as NodePort of 30090 by default. 
The port is used for client image classification requests.
```
is this correct?